tests/kamereon/test_kamereon_vehicle_data.py

```diff
@@ -304,7 +304,6 @@ def test_location_v1() -> None:
     assert vehicle_data.gpsLongitude == 11.1234567
     assert vehicle_data.lastUpdateTime == "2020-02-18T16:58:38Z"
 
-
```

Comment:
This carriage return needs to come back

Reply:
I think I got lost in those carriage returns. Are they ok now?

Reply:
Almost ready: just this one carriage return that needs to be added back in.
I cannot add it a suggestion because it's a deletion.

Reply:
I added a CR, hopefully the right one